http-gateway/web/src/common/styles/default.scss

```diff
@@ -0,0 +1,30 @@
+@import './colors.scss';
+@import './sizes.scss';
+
+html,
+body,
+input,
+textarea,
+select {
+  font-family: 'Muli', sans-serif;

```

Comment:
Muli is a licensed adobe font and a "free" font from font squirrel?

Reply:
It appears so, I will investigate further.